### PR TITLE
Fix Top Bar Overflow #1561

### LINKED ISSUE #1561: Overflow Issue on Events Page
**Describe the bug**
Overflow in the top bar of Events Screen

**To Reproduce**

**Screenshots**
photo_2023-02-24_15-28-27

### PR DESCRIPTION
**Pull request by mmjsmohit**
**What kind of change does this PR introduce?**
Fixes overflow issue on top bar in `explore_events.dart`

**Issue Number:**

Fixes #1561 

**Did you add tests for your changes?**
Yes, added a widget test in `explore_events_test.dart` named `Testing if the top bar overflows`

**Snapshots/Videos:**

photo_2023-02-25_11-18-31

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by mmjsmohit** [comment]
Can this PR be reviewed now? @palisadoes @noman2002 @TheHazeEffect

File: lib/views/after_auth_screens/events/explore_events.dart [hunk 1]
```diff
@@ -84,85 +84,91 @@ class ExploreEvents extends StatelessWidget {
...
-                                            key: Key('ExploreEvents'),
-                                          );
-                                        },
-                                      );
-                                    },
-                                    child: Card(
-                                      key: homeModel?.keySEDateFilter,
+                              SingleChildScrollView(
+                                key: const Key("ExploreEventsTopBar"),
+                                scrollDirection: Axis.horizontal,
```
Comment (noman2002) [inline lib/views/after_auth_screens/events/explore_events.dart:89]: Remove this. We don't want it to be scrollable.

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by Ayush0Chaudhary** [comment]
@noman2002 @palisadoes can i continue this, if @mmjsmohit is not working on this anymore

**Comment by palisadoes** [comment]
@mmjsmohit Are you still working on this. Please respond within the next day or we will close the PR and remove your assignment from the issue

**Comment by mmjsmohit** [comment]
Yes I am working on this. Will try to make a new commit tomorrow.

**Comment by palisadoes** [comment]
We have a new check in place. All Talawa mobile app methods, classes and functions need to have suitable comments. Please refer to this issue as  reference for what is now required. The aim is to help us improve the quality of our automated documentation on docs.talawa.io.
 
- https://github.com/PalisadoesFoundation/talawa/issues/1479

To fix this issue:

1. You will be required to remove a line containing `ignore_for_file`
1. After doing so you'll be requested to update the comments appropriately

We have also tried to make the documentation step automatically update your IDE with a custom Linter. If you have any comments, please contact @literalEval who worked on this code.

**Comment by literalEval** [comment]
@mmjsmohit try merging latest talawa/develop into your branch. That will fix the failing test case.

**Comment by noman2002** [comment]
@mmjsmohit Any progress ?

**Pull request closed without merging**